cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServletUtils.java

```diff
@@ -48,10 +47,10 @@ static String getProposalSummary(GoalOptimizer.OptimizerResult result) {
     int numLeaderMovements = 0;
     long dataToMove = 0;
     for (ExecutionProposal p : result.goalProposals()) {
-      if (p.actionType() == ActionType.REPLICA_MOVEMENT) {
+      if (!p.replicasToAdd().isEmpty() || !p.replicasToRemove().isEmpty()) {
```

Comment:
Do we consider a newly added replica's bytes as `dataToMove`? Should we clarify this with a comment line?

Reply:
replicasToAdd() includes all the replicas that do not exist in the old replica list. The a newly added replica is already included in the dataToMove.